src/python/doc/ls_simplex_tree_tflow_itf_ref.rst

```diff
@@ -0,0 +1,64 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+TensorFlow layer for lower-star persistence on simplex trees
+############################################################
+
+.. include:: differentiation_sum.inc
+
+Example of gradient computed from lower-star filtration of a simplex tree
+-------------------------------------------------------------------------
+
+.. testcode::
+
+    from gudhi.tensorflow import LowerStarSimplexTreeLayer
+    import tensorflow as tf
+    import gudhi as gd
+
+    st = gd.SimplexTree()
+    st.insert([0])
+    st.insert([1]) 
+    st.insert([2]) 
+    st.insert([3]) 
+    st.insert([4]) 
+    st.insert([5]) 
+    st.insert([6]) 
+    st.insert([7]) 
+    st.insert([8]) 
+    st.insert([9]) 
+    st.insert([10]) 
```

Comment:
We could skip the vertex insertions, they are implicitly added by the edges below.

Reply:
Thanks! Done.